docs/reference/search/render-search-template-api.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[render-search-template-api]]
+=== Render search template API
+++++
+<titleabbrev>Render search template</titleabbrev>
+++++
+
+Renders a <<search-template,search
+template>> as a <<search-search,search request body>>.
+
+////
+[source,console]
+----
+PUT _scripts/my-search-template
+{
+  "script": {
+    "lang": "mustache",
+    "source": {
+      "query": {
+        "match": {
+          "message": "{{query_string}}"
+        }
+      },
+      "from": "{{from}}",
+      "size": "{{size}}"
+    },
+    "params": {
+      "query_string": "My query string"
+    }
+  }
+}
+----
+// TESTSETUP
+////
+
+[source,console]
+----
+POST _render/template
+{
+  "id": "my-search-template",
+  "params": {
+    "query_string": "hello world",
+    "from": 20,
+    "size": 10
+  }
+}
+----
+
+[[render-search-template-api-request]]
+==== {api-request-title}
+
+`GET _render/template`
+
+`GET _render/template/<template-id>`
+
+`POST _render/template`
+
+`POST _render/template/<template-id>`
+
+[[render-search-template-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `read`
+<<privileges-list-indices,index privilege>> for at least one index pattern.
+
+[[render-search-template-api-path-params]]
+==== {api-path-parms-title}
+
+`<template-id>`::
+(Optional, string) ID of the search template to render. If no `source` is
+specified, this or the `id` request body parameter is required.
+
+[[render-search-template-api-request-body]]
+==== {api-request-body-title}
+
+`id`::
+(Required*, string) ID of the search template to render. If no `source` is
```

Comment:
I stumbled across the "*" syntax here, but I saw this is used elsewhere already. Just wanted to let you know I didn't really know what to make of it at first, but when I read both parameter entries it makes sense.